CHANGELOG.md

```diff
@@ -2,9 +2,14 @@
 
 ## Unreleased
 
+### Features
+
+- instance: add a protection flag to exo compute #608
+
 ### Improvements
 
 - go.mk: update to v2.0.2 #630 
+- instance: add a protection flag to exo compute #608
```

Comment:
you don't need to add it twice, just once under features.

Reply:
was by mistake.